src/pages/on-chain-governance/shared/getTotalVotes.ts

```diff
@@ -1,20 +1,21 @@
+import BigNumber from "bignumber.js";
+
 export function getVotePercentage(
-  percYes: number,
-  percNo: number,
-  percNeutral: number,
-  voteDecision: string
+  numYes: number,
+  numNo: number,
+  numNeutral: number
 ) {
-  const totalVotes = percYes + percNo + percNeutral;
-  switch (voteDecision) {
-    case "yes":
-      return Math.round((percYes / totalVotes) * 100);
-      break;
-    case "no":
-      return Math.round((percNo / totalVotes) * 100);
-      break;
+  const totalVotes = numYes + numNo + numNeutral;
+
+  const percYes = new BigNumber((numYes / totalVotes) * 100).precision(2);
```

Comment:
lol noooo, use operator from bignumber `.div()` and `.mul()`

Reply:
updated